src/datagen/main/java/mekanism/common/recipe/impl/ControlCircuitRecipeProvider.java

```diff
@@ -34,6 +35,11 @@ public void addRecipes(RecipeOutput consumer, HolderLookup.Provider registries)
         addCircuitUpgradeRecipe(consumer, MekanismItems.ADVANCED_CONTROL_CIRCUIT, MekanismTags.Items.CIRCUITS_BASIC, MekanismTags.Items.ALLOYS_INFUSED, basePath, "advanced");
         addCircuitUpgradeRecipe(consumer, MekanismItems.ELITE_CONTROL_CIRCUIT, MekanismTags.Items.CIRCUITS_ADVANCED, MekanismTags.Items.ALLOYS_REINFORCED, basePath, "elite");
         addCircuitUpgradeRecipe(consumer, MekanismItems.ULTIMATE_CONTROL_CIRCUIT, MekanismTags.Items.CIRCUITS_ELITE, MekanismTags.Items.ALLOYS_ATOMIC, basePath, "ultimate");
+
+        //infusion variants that save on the base ingots needed and half an alloy
```

Comment:
Is this comment about "half an alloy" correct? I feel like it is no longer correct.

Reply:
Whoops